File: mo-system/mo-system-service/src/main/java/ink/ckx/mo/system/config/PasswordEncoderConfig.java
package ink.ckx.mo.system.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
import org.springframework.security.crypto.password.PasswordEncoder;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/17
 */
@Configuration
public class PasswordEncoderConfig {

    @Bean
    public PasswordEncoder passwordEncoder() {
        return new BCryptPasswordEncoder();
    }
}

File: mo-admin/mo-admin-service/src/main/kotlin/ink/ckx/mo/admin/config/PasswordEncoderConfig.kt
package ink.ckx.mo.admin.config

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
import org.springframework.security.crypto.password.PasswordEncoder

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/17
 */
@Configuration
class PasswordEncoderConfig {

    @Bean
    fun passwordEncoder(): PasswordEncoder {
        return BCryptPasswordEncoder()
    }
}